opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -472,17 +473,21 @@ def create_span(
         return INVALID_SPAN
 
     @contextmanager  # type: ignore
-    def use_span(self, span: "Span") -> typing.Iterator[None]:
+    def use_span(
+        self, span: "Span", end_on_exit: bool = False
+    ) -> typing.Iterator[None]:
         """Context manager for controlling a span's lifetime.
 
-        Start the given span and set it as the current span in this tracer's
-        context.
+        Set the given span as the current span in this tracer's context.
 
-        On exiting the context manager stop the span and set its parent as the
-        current span.
+        On exiting the context manager set the span that was previously active
+        as the current span (this is usually but not necessarily the parent of
+        the given span). If ``end_on_exit`` is ``True``, then the span is also
+        ended when exiting the context manager.
 
         Args:
             span: The span to start and make current.
+            end_on_exit: Whether to end the span automatically when leaving the context manager.
```

Comment:
So black doesn't care about long lines in docstrings and we disabled the line length check in favor of black?

Reply:
```suggestion
            end_on_exit: Whether to end the span automatically when leaving the
                context manager.
```